pyproject.toml

```diff
@@ -37,7 +37,9 @@ pyyaml = "^6.0.1"
 boto3 = "^1.26.75"
 jsonschema = "^4.19.1"
 lightkube = "^0.15.2"
-psycopg2 = "^2.9.9"
+psycopg2-binary = "^2.9.9"
```

Comment:
question: Why do we replace `psycopg2` with `psycopg2-binary` here? AFAIK, Postgres recommends `psycopg2` be built from source in place of `psycopg2-binary`.